src/main/java/com/depromeet/domain/member/domain/Member.java

```diff
@@ -115,6 +119,10 @@ public void register(String nickname) {
         this.role = MemberRole.USER;
     }
 
+    public void withdrawal(MemberStatus memberStatus) {
+        this.status = memberStatus;
+    }
```

Comment:
파라미터 지우고 이미 탈퇴한 경우 예외 추가해도 좋을 것 같아용
```suggestion
    public void withdrawal() {
        if (this.status == MemberStatus. DELETED) {
            throw ...
        }
        this.status = MemberStatus. DELETED;
    }
```